thumbor/src/thumbor_/thumbor.conf

```diff
@@ -114,19 +114,25 @@ UPLOAD_DEFAULT_FILENAME = "image"
 ## them again with the loader
 # STORAGE = "thumbor.storages.redis_storage"
 # STORAGE = "thumbor.storages.no_storage"
+
 STORAGE = "thumbor.storages.file_storage"
 # STORAGE = "thumbor.storages.mixed_storage"
 # STORAGE = "thumbor.compatibility.storage"
 
+
+
+# Specify HTTP port through CLI args
+HTTP_ADDRESS = '0.0.0.0'
+
 ## root path of the file storage
-FILE_STORAGE_ROOT_PATH = join(home, "thumbor", "storage")
+FILE_STORAGE_ROOT_PATH = "/data/thumbor/storage"
```

Comment:
Can these config values be set at runtime with an environment variable? If so, perhaps we could leave the storage paths as the defaults, and only override them in the k8s pods?

Reply:
So this is possible, though we would have to further adapt the server:

def get_config(config_path, use_environment=False):
    if use_environment:
        Config.allow_environment_variables()

    lookup_paths = [os.curdir, expanduser("~"), "/etc/", dirname(__file__)]

    return Config.load(config_path, conf_name="thumbor.conf", lookup_paths=lookup_paths)

If we set this to true, thumbor will look for environment variables that match the config names and will overwrite them accordingly. 

The issue here is this will clash with importing thumbor.server as this is another modification to the default config (I imagine?)